test/models/user_test.rb

```diff
@@ -214,5 +214,28 @@ class UserTest < ActiveSupport::TestCase
       @user.reload
       assert @user.disabled_by_fund
     end
+
+    describe 'nightly cleanup task - disable_inactive_users' do
+      before do
+        @disabled_user = create(:user, last_sign_in_at: 10.months.ago)
+        @admin = create(:user, role: :admin, last_sign_in_at: 11.months.ago)
+        @no_login_user = create(:user, created_at: 12.months.ago)
+        @nondisabled_user = create(:user, last_sign_in_at: 8.months.ago)
+        User.disable_inactive_users
+      end
+
+      it 'should find inactive users with logins before cutoff and disable' do
+        [@disabled_user, @no_login_user].each do |user|
+          user.reload
+          assert user.disabled_by_fund?
+        end
+
+        # No locking admins or users with recent activity
+        [@admin, @nondisabled_user].each do |user|
+          user.reload
+          refute user.disabled_by_fund?
```

Comment:
Love the use of refute, but you could just do `assert !user.disabled_by_fund?` (pseudocode) probably, which might read easier to a newish Ruby person.

Reply:
Rubocop says we're both wrong as of 0.56: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Rails/RefuteMethods

```
Use `assert_not` methods instead of `refute` methods.
```

:(